ui/webui/Makefile.am

```diff
@@ -81,12 +81,13 @@ $(UPDATES_IMG):
 	test/prepare-updates-img
 
 integration-test: bots test/common $(UPDATES_IMG)
-	test/run-tests
+	test/run-tests --coverage
 
 # checkout Cockpit's test API; this has no API stability guarantee, so check out a stable tag
 # when you start a new project, use the latest release, and update it from time to time
+# FIXME: workaround for including code coverage support, not yet contained in a released cockpit version
 test/common:
-	git clone -b 266 --depth=1 https://github.com/cockpit-project/cockpit.git tmp/cockpit
+	git clone -b main --depth=1 https://github.com/cockpit-project/cockpit.git tmp/cockpit
```

Comment:
Is this an intentional change that should be part of the coverage PR ? As (if I understand this correctly) this is essentially unpinning the cockpit tooling version we use for tests I think it should be at least a separate commit.

Reply:
The issue is that testlib changes to suppor tcode coverage is not released yet in cockpit. See the comment above this line. 

This can be jsut bumped up to 267 next week that cockpit releases.